x-pack/plugins/secops/public/components/page/network/ip_overview/field_renderers.test.tsx

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+import AutonomousSystem = GetIpOverviewQuery.AutonomousSystem;
+import { mountWithIntl } from 'test_utils/enzyme_helpers';
+
+import { GetIpOverviewQuery, HostEcsFields, IpOverviewType } from '../../../../graphql/types';
+import { TestProviders } from '../../../../mock';
+
+import {
+  autonomousSystemRenderer,
+  dateRenderer,
+  hostNameRenderer,
+  locationRenderer,
+  whoisRenderer,
+} from './field_renderers';
+import { mockData } from './mock';
+
+describe('Field Renderers', () => {
+  describe('#locationRenderer', () => {
+    test('it renders correctly against snapshot', () => {
+      const wrapper = mount(
```

Comment:
I would use `shallow` for the snapshot. If the snapshot makes no sense or is still too large (> ~300 lines), then I would remove it.  Right now at a `mount` it is too large and github is trying to hide it from us.

Reply:
Thanks for calling attention to this. I started with shallow (as per team discussions about snapshot testing), but I didn't seem to be getting the data I wanted to test. Well turns out I was missing a field in the mock data, so doubly thanks! :) I updated the mock and moved over to shallow's for all snapshot tests.